src/main.rs

```diff
@@ -23,13 +25,28 @@ mod web;
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
-struct Args {
+pub struct Args {
     /// Override the port to host the web-ui and API on
     #[arg(short, long)]
     pub port: Option<u16>,
     /// Override the config file to use
     #[arg(short, long)]
     pub config: Option<String>,
+    /// Override the default tf2 directory
+    #[arg(short, long)]
+    pub tf2_dir: Option<String>,
+    /// Override the configured/default rcon password
+    #[arg(short, long)]
+    pub rcon_pword: Option<String>,
+    /// Override the configured Steam API key,
+    #[arg(short, long)]
+    pub api_key: Option<String>,
+    /// Whether to rewrite the user localconfig.vdf to append the new launch options,
+    #[arg(short = 'l')]
+    pub rewrite_launch_options: Option<bool>,
+    /// Whether to panic on detecting missing launch options
+    #[arg(short)]
+    pub no_panic_on_missing_launch_options: Option<bool>,
```

Comment:
These don't need to be options:
```rust
    /// Whether to rewrite the user localconfig.vdf to append the new launch options,
    #[arg(short = 'l', default_value_t = false)]
    pub rewrite_launch_options: bool,
    /// Whether to panic on detecting missing launch options
    #[arg(short, default_value_t = false)]
    pub no_panic_on_missing_launch_options: bool,
```

Reply:
bump

Reply:
Right - since they are now actuall proper set flags and not values, there is no semantic meaning to `None` versus `True/False`